server/backend/cron/news.go

```diff
@@ -9,6 +9,8 @@ import (
 	"strings"
 	"time"
 
+	"gorm.io/gorm/clause"
+
```

Comment:
Generally you just separate external from std imports and proto imports. You can just keep this in one block together with the github imports.

Reply:
The import grouping/formatting is done by [`goimports`](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) => if the current import formatting is bad, I would like to change it in another PR